build-external-api-jar/run.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-SOURCES_DIR="/tmp/idea-sources"
```

Comment:
This directory doesn't exist as idea is cloned by ide-probe and cleaned up after